### PR TITLE
Update portuguese Brazill (PT-BR)

### PR DESCRIPTION
**Pull request by V104-byte**
Grammatical errors corrected and some texts changed for better understanding in Portuguese Brazil

Others:

MY id in Nebulous (31016893)

### CODE REVIEW

**Review by AssaNebulous — COMMENTED** [review]
Hi, I don't think such a change is necessary.
Por favor desconsidere.

File: res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -378,7 +378,7 @@
     <string name="Clan_Warrior">"Guerreiro do Clã"</string>
     <string name="Win_a_Clan_War">"Ganhar uma Guerra de Clã"</string>
     <string name="Global">"Global"</string>
-    <string name="Spectating">"Espectador"</string>
+    <string name="Spectating">"Espectar um Jogador"</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:381]: Previous translation was correct

File: res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -289,8 +289,8 @@
     <string name="No_such_user_found_">"Nenhum usuário encontrado."</string>
     <string name="No_open_public_clans_exist_">"Não existem clãs públicos abertos."</string>
     <string name="You_aren_t_invited_to_that_clan_">"Você não está convidado para este clã."</string>
-    <string name="No_such_clan_exists_">"Não existe tal clã."</string>
-    <string name="That_clan_is_full_">"Esse clã está cheio."</string>
+    <string name="No_such_clan_exists_">"Este clã não existe."</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:292]: Previous translation was correct

File: res/values-pt-rBR/strings.xml [hunk 3]
```diff
@@ -68,13 +68,13 @@
     <string name="Difficulty_">"Dificuldade:"</string>
     <string name="Delay_Disconnect">"Atrasar a Desconexão"</string>
 
-    <string name="Max_Players">"Máx. de Jogadores"</string>
-    <string name="Min_Players">"Mín. de Jogadores"</string>
+    <string name="Max_Players">"Máximo de Jogadores"</string>
+    <string name="Min_Players">"Mínimo de Jogadores"</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:72]: Lines 71 and 72:

Please keep the abbreviations like it is in English (blue colour)

File: res/values-pt-rBR/strings.xml [hunk 4]
```diff
@@ -68,13 +68,13 @@
...
 
     <string name="OK">"OK"</string>
     <string name="CANCEL">"CANCELAR"</string>
     <string name="Chat_Name">"Nome do Chat"</string>
-    <string name="Max_People">"Máx. de Pessoas"</string>
+    <string name="Max_People">"Limite de Pessoas"</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:77]: Please keep the abbreviation here; previous translation was correct

File: res/values-pt-rBR/strings.xml [hunk 5]
```diff
@@ -451,8 +451,8 @@
 
 
     <string name="Copy_ID">"Copiar ID"</string>
-    <string name="View_Profile">"Ver Perfil"</string>
-    <string name="Send_Message">"Enviar Mensagem"</string>
+    <string name="View_Profile">"Visualizar Perfil"</string>
+    <string name="Send_Message">"Enviar Uma Mensagem"</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:455]: Line 454: Previous translation was correct
Line 455: Inserting "uma" will make the string length too long. Previous translation was correct

File: res/values-pt-rBR/strings.xml [hunk 6]
```diff
@@ -471,7 +471,7 @@
     <string name="ENTER">"ENTRAR"</string>
     <string name="_1v1">"1v1"</string>
     <string name="Practice">"Praticar"</string>
-    <string name="ArenaDescription">Você terá apenas uma vida. O vencedor receberá %s plasma. Negociar vitórias ou "pontos gratuitos" resultará em um banimento.</string>
+    <string name="ArenaDescription">Você terá apenas uma vida. O vencedor receberá %s plasma. Negociar vitórias ou "pontos gratuitos" resultará em banimento.</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:474]: Previous translation was correct

File: res/values-pt-rBR/strings.xml [hunk 7]
```diff
@@ -566,7 +566,7 @@
     <string name="Remove_Clan_Member">"Remover Membro do Clã"</string>
     <string name="Flag_lobby_as_inappropriate">"Denunciar sala como inapropriada "</string>
     <string name="Promote_Clan_Member">"Promover Membro do Clã"</string>
-    <string name="Leave_Clan">"Sair do Clã"</string>
+    <string name="Leave_Clan">"Sair deste Clã"</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:569]: Previous translation was correct (based on English)

File: res/values-pt-rBR/strings.xml [hunk 8]
```diff
@@ -729,8 +729,8 @@
     <string name="upload_warning">"Tem certeza de que deseja enviar esta skin? Outros jogadores serão capazes de ver esta skin. Pode levar até uma semana para a sua skin ser revisada. Skins inapropriadas serão rejeitadas e não serão reembolsadas!"</string>
 
     <string name="In_Review">"Em Revisão"</string>
-    <string name="Account_Uploads">"Envios para a Conta"</string>
-    <string name="Clan_Uploads">"Envios para o Clã"</string>
+    <string name="Account_Uploads">"Suas Skins Enviadas"</string>
+    <string name="Clan_Uploads">"Skins Do Clã Enviadas"</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:733]: Lines 732 and 733: Previous translations were correct

File: res/values-pt-rBR/strings.xml [hunk 9]
```diff
@@ -756,22 +756,22 @@
...
+    <string name="Nebulous_needs_Storage_permissions_to_backup_your_single_player_progress_">Nebulous precisa de permissão de Armazenamento para salvar seu progresso de único jogador!</string>
     <string name="Nebulous_needs_Coarse_Location_permissions_to_find_nearby_Bluetooth_devices_">Nebulous precisa de permissão de Localização para encontrar dispositivos Bluetooth próximos.</string>
     <string name="FFA_ULTRA">FFA ULTRA</string>
     <string name="FFAU">FFAU</string>
     <string name="_in_a_FFAU_Game"> em um jogo de FFA ULTRA</string>
 
     <string name="BANNED_PLAYERS">JOGADORES BANIDOS</string>
     <string name="MAIL">E-MAIL</string>
-    <string name="WRITE_MAIL">ESCREVER MENSAGEM</string>
+    <string name="WRITE_MAIL">ESCREVER E-MAIL</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:767]: Previous translation was correct

File: res/values-pt-rBR/strings.xml [hunk 10]
```diff
@@ -756,22 +756,22 @@
...
     <string name="Days">Dias</string>
     <string name="Message">Mensagem</string>
     <string name="Subject">Assunto</string>
     <string name="SEND">ENVIAR</string>
     <string name="To">Para</string>
     <string name="RESET">RESTAURAR</string>
-    <string name="Message_Sent_">Mensagem Enviada!</string>
+    <string name="Message_Sent_">E-mail Enviado!</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:774]: Previous translation was correct

File: res/values-pt-rBR/strings.xml [hunk 11]
```diff
@@ -780,7 +780,7 @@
     <string name="DELETE">EXCLUIR</string>
     <string name="Note__You_can_unblock_someone_by_sending_them_a_friend_request_">Nota: Você pode desbloquear alguém, enviando-lhes um pedido de amizade.</string>
     <string name="Select_Friend">Selecionar Amigo</string>
-    <string name="Failed_to_send_mail_">Falha ao enviar a mensagem.</string>
+    <string name="Failed_to_send_mail_">Falha ao enviar o e-mail.</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:783]: Previous translation was correct

**Review by Jeii20 — CHANGES_REQUESTED** [review]

**Comment by Jeii20** [comment]
PR closed. Too many invalid changes. You tried to correct what was already correct at lines 71, 72,  77, 292, 381, 454, 455, 474, 569, 732, 733, 767, 774 and 783.

**Pull request closed without merging**